package.json

```diff
@@ -2,9 +2,9 @@
   "name": "@theia/node-pty",
   "description": "Fork pseudoterminals in Node.JS",
   "author": {
-    "name": "Daniel Imms"
+    "name": "Microsoft Corporation"
   },
-  "version": "0.7.8-theia004",
+  "version": "0.8.1-theia005",
```

Comment:
I think the `005` part is so we can release more than once per upstream release, if needed. Should probably be reset to `001` for this new `0.8.1` release?

Reply:
I prefer it only going up each time we will need to make our own release, easier to track IMO.

Reply:
I think we can even improve the format by doing `x.x.x-theia.y`, then it would work like the `next` releases for Theia, except this time we should be able to refer to it as `theia` releases. Will try that. Already played with https://semver.npmjs.com/ against `@theia/node-pty`, and I think we can improve it.